build_scripts/build_usd.py

```diff
@@ -726,24 +780,31 @@ def InstallBoost_Helper(context, force, buildArgs):
                                              dontExtract=dontExtract)):
         bootstrap = "bootstrap.bat" if Windows() else "./bootstrap.sh"
 
+        if context.targetIos:
```

Comment:
there's no link dependency on boost from iOS, because there's no support for python bindings to USD on iOS, right?

Can we simplify things by merely unarchiving boost, and pointing the iOS build at the headers, and calling it a day? In the unlikely event that one of the boost headers needs configuration for iOS, maybe we can just patch that header. But to the best of my recollection, even that shouldn't be necessary.

Reply:
That worked really well! Thank you so much!